applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1022,7 +1023,8 @@ void SmallStrainUPwDiffOrderElement::CalculateOnIntegrationPoints(const Variable
                     BodyAcceleration[idim] += Variables.Nu[i] * Variables.BodyAcceleration[Index++];
             }
 
-            CalculateFluidPressure(Variables);
+            Variables.FluidPressure =
+                RetentionLaw::Parameters::CalculateFluidPressure(Variables.Np, Variables.PressureVector);
```

Comment:
I have fixed the missing assignment a few minutes ago on `master` by merging one of my pending PRs. Please update this branch by merging `master` into it (you will probably need to a resolve a merge conflict, since we have touched the same lines).

Reply:
You are right there were merge conflicts.